class NotFoundError(Exception):
    pass

class NotStoredError(Exception):
    pass

